app/services/billable_metrics/prorated_aggregations/sum_service.rb

```diff
@@ -32,6 +32,23 @@ def aggregate(options: {})
         result.service_failure!(code: 'aggregation_failure', message: e.message)
       end
 
+      def compute_per_event_prorated_aggregation
+        persisted_events = persisted_query
+          .pluck(
+            Arel.sql("(COALESCE((#{sanitized_field_name})::numeric, 0)) * (#{persisted_pro_rata})::numeric"),
+          )
+
+        period_events = period_query
+          .pluck(
+            Arel.sql(
+              "(COALESCE((#{sanitized_field_name})::numeric, 0)) * "\
+              "(#{duration_ratio_sql('events.timestamp', to_datetime)})::numeric"
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)